doc/radosgw/cloud-restore.rst

```diff
@@ -142,121 +159,133 @@ For example:
+----------------------------
 
-Examples to Restore Object:
----------------------------
-
-Using S3 RestoreObject CLI
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Below options of `S3 restore-object <https://docs.aws.amazon.com/cli/latest/reference/s3api/restore-object.html>`__ CLI are supported  -
-
-
-Syntax
-
-::
+Using the S3 RestoreObject CLI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+These options to the `S3 restore-object <https://docs.aws.amazon.com/cli/latest/reference/s3api/restore-object.html>`
+CLI are supported:
 
+```
   $ aws s3api restore-object 
             --bucket <value>
             --key <value>
             [--version-id <value>]
             --restore-request (structure) {
               Days=<integer>
             }
+```
 
-Note: ``Days`` is optional and if not provided, the object is restored permanently
+Note: ``Days`` is optional and if not provided, the object is restored permanently.
 
 Example 1:
 
-::
-
+```
   $ aws s3api restore-object  --bucket bucket1 --key doc1.rtf 
                               [--version-id 3sL4kqtJlcpXroDTDmJ+rmSpXd3dIbrHY+MTRCxf3vjVBH40Nr8X8gdRQBpUMLUo]
                               --restore-request Days=10 
                               ....
+```
 
-This will restore the object `doc1.rtf` of the given version for the duration of 10 days.
-
-
+This will restore the object ``doc1.rtf`` at an optional version,
+for the duration of 10 days.
 
 Example 2:
 
-::
-
+```
   $ aws s3api restore-object  --bucket bucket1 --key doc2.rtf --restore-request {} ....
+```
 
-This will restore the object `doc2.rtf` permanently and will be treated as regular object.
 
+This will restore the object ``doc2.rtf`` permanently and it will be treated as regular object.
 
 
-Using S3 GetObject CLI
-~~~~~~~~~~~~~~~~~~~~~
-Ensure ``allow_read_through`` tier-config option is enabled.
+Using the S3 GetObject CLI
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Example 3:
+Ensure that the ``allow_read_through`` tier-config option is enabled.
 
-::
+Example 3:
 
+```
   $ aws s3api get-object  --bucket bucket1 --key doc3.rtf ....
+```
 
-This will restore the object `doc3.rtf` for the duration of the ``read_through_restore_days`` configured.
-
+This will restore the object `doc3.rtf`` for ``read_through_restore_days`` days.
 
-Note: The above CLI command may time out if the object restoration takes too long. Before reissuing the command, you can verify the restoration status.
+Note: The above CLI command may time out if object restoration takes too long.
+You can verify the restore status before reissuing the command.
 
 
 Verifying the restore status
 ----------------------------
-Verify the status of the restore by running an `S3 HeadObject <https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadObject.html#API_HeadObject_ResponseSyntax>`__  request. The response includes ``x-amz-restore`` header if either the object restoration is in progress or a copy of it is already restored.
+Verify the status of the restore by issuing
```

Comment:
Maybe I'm just being pedantic, but it seems to me that "the restore" here is not quite as much of a noun as "the restoration" is. I know that it is the fashion in this industry to use words like the bare infinitive "compute" as a noun, but I regard this as a sign of the decline of the understanding of the English language and I want to resist it as much as possible.

Reply:
pushed